packages/desktop-client/package.json

```diff
@@ -7,7 +7,7 @@
   ],
   "devDependencies": {
     "@juggle/resize-observer": "^3.4.0",
-    "@playwright/test": "^1.41.1",
+    "@playwright/test": "1.41.1",
```

Comment:
We use this image on the build process: image: mcr.microsoft.com/playwright:v1.41.1-jammy

It's locked in at version 1.41.1 - adjusting here so we're always aligned